Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -285,76 +285,76 @@ void EnhancementsWidget::AddDescriptions()
   static const char TR_INTERNAL_RESOLUTION_DESCRIPTION[] =
       QT_TR_NOOP("Specifies the resolution used to render at. A high resolution greatly improves "
```

Comment:
>Specifies the resolution used to render at.

I'm still surprised you didn't change this one! Might I suggest:

>Controls the rendering resolution of the emulated software.

Reply:
Great idea, what do you think about just doing "Controls the rendering resolution" for conciseness?

Reply:
Well, I guess that's fine, but I wanted to make it clear what it is controlling. Up to you!